packages/cli/cloud/src/logout/action.ts

```diff
@@ -11,9 +13,26 @@ export default async (ctx: CLIContext) => {
     return;
   }
   const cloudApiService = await cloudApiFactory(token);
+  const config = await cloudApiService.config();
```

Comment:
I think we missed it in the original dev of this action, but could you add a try catch around all awaits to not end up with an uncatched failed promise? 🤔

Reply:
We could have the error within cloudApiService's `config` function? we could even extract .data to simplify the returned value.

Current code:

```
    config(): Promise<AxiosResponse<CloudCliConfig>> {
      return axiosCloudAPI.get('/config');
    },
```

Something like the following:
```
async config(): Promise<CloudCliConfig> {
  try {
    const response = await axiosCloudAPI.get('/config');
    const cloudCliConfig = response.data;

    if (!cloudCliConfig) {
      throw new Error('Undefined cloudCliConfig');
    }

    return cloudCliConfig;
  } catch (error) {
    throw new Error(`Couldn't retrieve the cloud CLI config - ${error.message}`);
  }
},
```
I'll go with that, I think it'll be more convenient and it's still acceptable logic for an API service. I don't think we need to whole Axios response anywhere.

Reply:
I think it's better to stick with the full Axios response, as it is used in other API functions calls (e.g. project creation), and staying coherent.
It's ok to still handle the error in the action, I believe.

Reply:
Also we want to hide technical errors to our users and only display business error messages + debug log the technical one.

Reply:
Modifications done!

I added the try catch within config() and kept the AxiosResponse signature.
(Passed ctx through the factory. The whole axios response will be displayed in debug mode if an error occurs)